grafana-plugin/src/components/IntegrationCollapsibleTreeView/IntegrationCollapsibleTreeView.tsx

```diff
@@ -34,7 +41,12 @@ const IntegrationCollapsibleTreeView: React.FC<IntegrationCollapsibleTreeViewPro
   }, [configElements]);
 
   return (
-    <div className={cx('integrationTree__container')}>
+    <div
+      className={cx(
+        'integrationTree__container',
+        customContainerClassName ? `integrationTree__container--${customContainerClassName}` : ''
```

Comment:
Why do we need `integrationTree__container--` prefix for class provided as a prop? Maybe better just to pass`customContainerClassName` as is and allow adding custom classname while using this component?

Reply:
I guess because I wanted the class to be part of the TreeView component and not from the outside, this way I keep all the display changes private to that scss file. Like I didn't want option to provide a class from the outside because it should be handled by the scss part of the component

Reply:
With this implementation every time user of IntegrationCollapsibleTreeView component wants to apply styles provided by `.integrationTree__container--timeline-view` class, needs to now exactly that he/she needs to pass `customContainerClassName="timeline-view"`. This is not very handy and is error-prone, any typo in this string will cause a bug. Also when clas`integrationTree__container--timeline-view` is renamed in future, there will be no indication from Typescript that there are places where wrong class name is used.

I suggest one of 2 solutions:
1) Since `customContainerClassName` has only one usage, we can just create cx('timeline-view') class in `ExpandedIntegrationRouteDisplay.module.scss` and add there 
```
{
  margin-left: -19px;
  border: 0 !important;
  width: 4px;
  background-color: var(--secondary-background);
}
```
2) If we believe that it's not a single case and in future we will have more components that will want to apply the same styles, we can change `customContainerClassName: string` prop into `isTimelineView: boolean` in `IntegrationCollapsibleTreeView`. Then apply the class conditionally based on this class:
```
<div
      className={cx(
        'integrationTree__container',
        { `integrationTree__container--timeline-view`: prop.isTimelineView }
      )}
    >
```